src/core.c/Exception.pm6

```diff
@@ -343,7 +343,7 @@ my class X::Pragma::NoArgs is Exception {
 }
 my class X::Pragma::CannotPrecomp is Exception {
     has $.what = 'This compilation unit';
-    method message { "$.what may not be pre-compiled" }
+    method message { "$.what cannot be pre-compiled and thus cannot be used in a module" }
```

Comment:
While the case in question (about `use lib`) results in a sensible hint, I wonder if other values of `$!what` end up more confusing? For example, given the default `what`, it'd say "This compilation unit cannot be pre-compiled  and thus cannot be used in a module", which is a bit odd.

Reply:
Hm, looking at that one case, it seems you'd pretty much have to have invoked `raku --target=mbc --output=foo.moarvm something-with-no-precompilation.rakumod` in order to see that message, which...most probably nobody does, since we manage precompilation automatically. If a tree falls in the forest and nobody hears it... :-)